test/helpers/playerinteractionwrapper.js

```diff
@@ -634,6 +634,11 @@ class PlayerInteractionWrapper {
         return card;
     }
 
+    scrap(card) {
```

Comment:
A minor thing: I would have called it "discard". Scrap is the effect from discarding a card. I think "discard" reflects the action the player is doing better.

Reply:
I agree, but you're the one who suggested calling it `scrap` in your original comment!  Oh well.  I might make an MR later cleaning some of this testing stuff up.

Reply:
oh man 🤦 sorry abou that 😞